nexus_allowlist/actions.py

```diff
@@ -185,6 +209,44 @@ def recreate_privileges(
     )
     cran_privilege_names.append(privilege_name)
 
+    # Content selector and privilege for APT 'Packages.gz' file which contains an
+    # metadata for all archived packages
+    privilege_name = create_content_selector_and_privilege(
+        nexus_api,
+        name="apt-packages",
+        description="Allow access to 'Packages.gz' file in APT repository",
+        expression=f'format == "apt" and path=~"^/dists/{APT_DISTRO}/.*/Packages.gz"',
```

Comment:
Does this work for all apt repositories? Debian, Ubuntu, Linux Mint for example.

Reply:
It seems to match for ubuntu `https://fr.archive.ubuntu.com/ubuntu/dists/bionic/main/binary-amd64/Packages.gz`. I infer it's a standard from apt/debian.